### PR TITLE
[Milkman] Create empty home page

### PR DESCRIPTION
**Pull request by luizakp**
This PR:
- creates network routes
- move formatNumber to utils
- move gql Query to top server component
- created client home page with orders table and create new order button following figma

https://www.loom.com/share/3bde4d7bebd44a24b5289d85951b7206?sid=e8763614-b846-41e5-b457-9beb4956378f

### CODE REVIEW

File: apps/cow-tools/src/app/milkman/layout.tsx [hunk 1]
```diff
@@ -13,17 +15,22 @@ export const metadata: Metadata = {
...
 
-      <div className="sm:flex flex-1 gap-x-8 px-4 pt-4 text-white">
-        {children}
+        <div className="flex flex-1 overflow-auto p-4 sm:flex-row sm:gap-x-8 text-white">
+          {children}
+        </div>
+        <Footer
+          githubLink="https://github.com/bleu-fi"
+          discordLink="https://discord.gg/Z78vQvmHN2"
+        />
```
Comment (yvesfracari) [inline apps/cow-tools/src/app/milkman/layout.tsx:32]: All right, you moved the footer from another layout to here, right? Sounds good for me

File: apps/cow-tools/src/app/milkman/page.tsx [hunk 2]
```diff
@@ -1,17 +1,5 @@
...
-
-export default async function Page() {
-  return (
-    <div className="flex flex-col w-full">
-      <AddressComponentWithNoSSR />
-      <OrdersTable />
-    </div>
-  );
+export default function Page() {
+  redirect("/milkman/goerli");
```
Comment (yvesfracari) [inline apps/cow-tools/src/app/milkman/page.tsx:4]: 👍 
Can you create a task to change this at the end of the project?

Comment (yvesfracari) [reply apps/cow-tools/src/app/milkman/page.tsx:4]: Then link the issue here with a comment

**Review by yvesfracari — APPROVED** [review]
Great work @luizakp ! Thanks for taking the orders table as well at this task, I wasn't expecting just the "No orders for this user" text on this text hahahah

Would just double check the prettier changes on the balancer related files.

**Comment by luizakp** [comment]
> Great work @luizakp ! Thanks for taking the orders table as well at this task, I wasn't expecting just the "No orders for this user" text on this text hahahah
> 
> Would just double check the prettier changes on the balancer related files.

opsss my bad kkkkk understood wrong xD

File: apps/cow-tools/src/app/milkman/page.tsx (hunk 2 above)
Comment (luizakp) [reply apps/cow-tools/src/app/milkman/page.tsx:4]: here: https://linear.app/bleu-llc/issue/BLEU-366/change-default-home-page

File: apps/cow-tools/src/app/milkman/layout.tsx (hunk 1 above)
Comment (luizakp) [reply apps/cow-tools/src/app/milkman/layout.tsx:32]: https://github.com/bleu-fi/balancer-tools/pull/449#discussion_r1376076336

File: apps/cow-tools/src/app/layout.tsx [hunk 3]
```diff
@@ -13,10 +12,6 @@ export default function Layout({ children }: { children: React.ReactNode }) {
       <Fonts />
       <body className="bg-blue2 flex h-full flex-col">
         <RootLayout>{children}</RootLayout>
-        <Footer
-          githubLink="https://github.com/bleu-fi"
-          discordLink="https://discord.gg/Z78vQvmHN2"
-        />
```
[the remark this answers is not in the document]
Comment (luizakp) [reply apps/cow-tools/src/app/layout.tsx:19]: https://github.com/bleu-fi/balancer-tools/pull/449#discussion_r1376076336

**Comment by luizakp** [comment]
> Great work @luizakp ! Thanks for taking the orders table as well at this task, I wasn't expecting just the "No orders for this user" text on this text hahahah
> 
> Would just double check the prettier changes on the balancer related files.

I already ran pn format on the whole project and built both projects to see if everything is working

File: apps/cow-tools/src/components/Table.tsx [hunk 4]
```diff
@@ -0,0 +1,211 @@
+"use client";
```
Comment (ribeirojose) [inline apps/cow-tools/src/components/Table.tsx:1]: this is the kind of component that would be useful to have in a shared lib

File: apps/cow-tools/src/app/milkman/utils/type.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+export enum TransactionStatus {
+  TO_BE_EXECUTED = "To be executed",
+  MILKMAN_CREATED = "Milkman created",
+  ORDER_PLACED = "Order placed",
+  EXECUTING = "Executing",
+  EXECUTED = "Executed",
+  CANCELATION_TO_BE_EXECUTED = "Cancelation to be executed",
+  CANCELED = "Canceled",
+}
```
Comment (ribeirojose) [inline apps/cow-tools/src/app/milkman/utils/type.ts:9]: Will leave as a reminder, but I'd redo the whole tx logic. We can get inspo from Milkman's/Balancer/UNI UIs but can take a read at the code as well as they're all open source

File: apps/balancer-tools/src/app/apr/(components)/HomeOverviewCards.tsx [hunk 6]
```diff
@@ -1,5 +1,6 @@
+import { formatNumber } from "@bleu-balancer-tools/utils/formatNumber";
```
Comment (ribeirojose) [inline apps/balancer-tools/src/app/apr/(components)/HomeOverviewCards.tsx:1]: 👍  good refactor!

File: apps/cow-tools/src/app/milkman/[network]/HomePageWrapper.tsx
Comment (ribeirojose) [inline apps/cow-tools/src/app/milkman/[network]/HomePageWrapper.tsx:1]: `#would've` put this in `[network]/(components)/HomePageWrapper` or simply put this code into `[network]/page`

**Review by ribeirojose — COMMENTED** [review]
Looking 🔪 !

**Pull request merged**